index.html

```diff
@@ -250,7 +250,7 @@ <h3><a href="https://www.reddit.com/r/streamtheater/">Stream Theater</a></h3>
     <!--- Who We Are -->
     <div class="wrapper style1">
         <div class="title">Who We Are</div>
-        <section id="whoweare" class="container">
+        <section id="whoweare" class="container text-align-center">
```

Comment:
Adding a utility class `text-align-center` did the trick. I initially changed the id because it was already used and Lighthouse pinned me for that.